spec/lib/bank_accounts/account_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
+describe BankAccounts::Account do
+
+  it "responds to #new" do
+    expect(BankAccounts::Account).to respond_to(:new)
+  end
+
+  it "responds to #new with two arguments" do
+    expect(BankAccounts::Account).to respond_to(:new).with(2).arguments
+  end
+
+  let(:bob) { BankAccounts::Account.new(1, 1000) }
```

Comment:
Ha--Is there a reason you named this variable `bob`? Generally you'd probably want to use something that more accurately describes the object, so as boring as it may be, you might be better off just using `account` ;)